tests/test_ndarray.py

```diff
@@ -399,7 +399,8 @@ def test_getitem():
     b = numpy.ones((3))
     a = afnumpy.array(b)
     iassert(a[1:4],b[1:4])
-    iassert(a[3::-1],b[3::-1])
+    # This no longer works with current version of arrayfire, accessing a d_array with slice(None,-1,-1) does not give the expected output
+    #iassert(a[3::-1],b[3::-1]) 
```

Comment:
My previous comment was related to this line.

Reply:
done